### DISCUSSION TITLE
Lesson 1 - creation of SimpleStorage pending...

### DISCUSSION THREAD

**Discussion by mmarovich** [smartcontractkit/full-blockchain-solidity-course-py#1351] [Q&A] [answered]
Finished first contract function, but instead of deploying properly, the console simply hangs with a message stating: "creation of SimpleStorage pending..."  Code looks exactly the same... what have I done wrong?

![Capture](https://user-images.githubusercontent.com/23124935/160714495-17647ed9-921a-4e71-8b80-49931e4d6db0.PNG)

**Comment by Noveleader** [comment, marked as answer]
Hey @mmarovich include ```SPDX``` identifier too. Add this at top of your ```contract``` : 
```// SPDX-License-Identifier: MIT ```

**Reply by mmarovich** [reply]
Thank you!

**Comment by cromewar** [comment]
Hello @mmarovich 
First of all @Noveleader is right you must add a license Identifier, but you will discover it later during the course.
For me is a problem with the browser you are using, as I cannot see anything wrong on your code I strongly suggest to try this on another browser.

**Reply by mmarovich** [reply]
Why do you strongly suggest I use another browser?  Adding the line suggested by Noveleader fixed it...

**Reply by cromewar** [reply]
Some times the browser cache mess up and remix does not behave as it should, but anyways I'm glad @Noveleader helped you

**Reply by Noveleader** [reply]
> Some times the browser cache mess up and remix does not behave as it should, but anyways I'm glad @Noveleader helped you

Yeah it does happens a lot with many websites.
